crates/matrix-sdk-crypto/src/store/locks.rs

```diff
@@ -68,6 +106,16 @@ pub struct CryptoStoreLock {
 }
 
 impl CryptoStoreLock {
+    /// Amount of time a lease of the lock should last, in milliseconds.
+    pub const LEASE_DURATION_MS: u32 = 2000;
```

Comment:
Could be 1000, or even 800, so that it cannot be really perceived by the app end-user?

Reply:
Fwiw: we chatted about it, and we'll try renew-every 50ms / lease-for 500ms: still a 1:10 ratio, and the lease should be closer to non-perceivable by the user (ideally we'd go under 300ms, but that seems more dangerous with respect to the scheduler being super busy).